introduction.md

```diff
@@ -0,0 +1 @@
+I'm am me. I can be no one else but me.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.